src/sparsezoo/v2/model_directory.py

```diff
@@ -149,6 +157,9 @@ def __init__(
         )
 
         super().__init__(files=files, name=name, path=path, url=url)
+        from sparsezoo.v2.integration_validation.validator import IntegrationValidator
```

Comment:
why do we import here?, if necessary to do so at runtime, let's add a comment for why